test/includes/pre/sys/stat.h

```diff
@@ -1,28 +0,0 @@
-#ifndef _GNU_SOURCE
-# define _GNU_SOURCE
-#endif
-#include <sys/stat.h>
-
-static const mode_t kS_IFMT = S_IFMT;
-static const mode_t kS_IFSOCK = S_IFSOCK;
```

Comment:
the tests expect the k-prefixed symbol, so e.g. `kS_IFSOCK` in the tests would need to be replaced by `S_IFSOCK`

Reply:
It would not work for `sys/stat.h` - the definitions are in `bits/types.h`, which is private.
So I've reverted this back to only removing `pre/uv.h`.